web-marketplace/src/lib/i18n/locales/es.po

```diff
@@ -1,1122 +1,1093 @@
 msgid ""
```

Comment:
please be careful we've translated ES content with AI so running i18n:extract overwrites it
we still need to discuss how we should handle such updates and conflicts but in the mean time, make sure to run also i18n:translate